test/pure-component-test.output.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var React = require('React');
+
+function render() {
+  return <div/>;
+}
+
+function A(props) {
```

Comment:
I still like the const + arrow function mode. If you wanna go crazy, how about making it an option for the hipster people? :)